CHANGELOG.rst

```diff
@@ -30,6 +30,9 @@ Changes:
 - ``structlog.get_context()`` allows you to retrieve the original context of a bound logger.
   `#266 <https://github.com/hynek/structlog/issues/266>`_,
 
+- ``ProcessorFormatter`` no longer uses exceptions for control flow, allowing
```

Comment:
We use [semantic newlines](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) and please use a fully qualified symbol for ProcessorFormatter.

Reply:
Semantic newlines 👍